components/collector/src/source_collectors/quality_time/missing_metrics.py

```diff
@@ -25,16 +25,20 @@ async def _get_source_responses(self, *urls: URL) -> SourceResponses:
     async def _parse_source_responses(self, responses: SourceResponses) -> SourceMeasurement:
         """Get the metric entities from the responses."""
         datamodel_response, reports_response = responses
-        data_model = await datamodel_response.json()
+        self.data_model = await datamodel_response.json()
```

Comment:
I'm surprised that the linter doesn't demand typing defined for this class instance var

Reply:
I assume it's because `json()` returns `Any` (https://docs.aiohttp.org/en/stable/_modules/aiohttp/client_reqrep.html#ClientResponse.json).